app/assets/javascripts/google-analytics-events.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Tracks custom events with Google Analytics.
+ * This application uses the google-analytics-rails gem to track general usage
+ * data. This file defines additional user interations to be tracked with Google
+ * Analtyics.
+ * See https://developers.google.com/analytics/devguides/collection/analyticsjs/events
+ */
```

Comment:
@anarchivist: I changed file name to `google-analytics-events.js` and added comment, above.